src/tests/functional/agenda/test_schedule_export.py

```diff
@@ -62,3 +63,73 @@ def test_feed_view(slot, client, schedule_schema):
     response = client.get(slot.submission.event.urls.feed)
     assert response.status_code == 200
     assert slot.submission.event.schedules.first().version in response.content.decode()
+
+
+@pytest.mark.django_db
+def test_html_export_event_required():
+    from django.core.management import call_command
+    with pytest.raises(CommandError) as excinfo:
+        call_command('export_schedule_html')
+
+    assert 'required' in str(excinfo)
+    assert 'event' in str(excinfo)
```

Comment:
If you are this precise in checking the error messages, you can just check the complete message directly, imo.